x-pack/plugins/remote_clusters/public/sections/remote_cluster_list/detail_panel/detail_panel.js

```diff
@@ -27,7 +27,7 @@ export class DetailPanelUi extends Component {
     isLoading: PropTypes.bool,
     remoteCluster: PropTypes.object,
     closeDetailPanel: PropTypes.func.isRequired,
-    remoteClusterName: PropTypes.string.isRequired,
+    clusterName: PropTypes.string.isRequired,
```

Comment:
to cut down on verbosity, I've renamed most data props that begin with `remoteCluster-` to `cluster-`

Reply:
Maybe worth renaming the `remoteCluster` prop to just `cluster` too in that case.